code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1128,15 +1129,25 @@ There are several things that need to be remembered:
 	if(update_icons)
 		update_icons()
 
+/mob/living/carbon/human/proc/get_tail_layer()

```

Comment:
No real reason for this to be a proc instead of a macro.

Reply:
I forgot about macroings existing.